scripts/install-appstudio-e2e-mode.sh

```diff
@@ -40,12 +47,10 @@ function initializeSPIVault() {
 
 # Secrets used by pipelines to push component containers to quay.io
 function createApplicationServiceSecrets() {
-    echo -e "[INFO] Creating application-service related secrets in $E2E_APPLICATIONS_NAMESPACE namespace"
+    echo -e "[INFO] Creating application-service related secrets in $SHARED_SECRET_NAMESPACE namespace"
 
     echo "$QUAY_TOKEN" | base64 --decode > docker.config
-    oc create namespace "$E2E_APPLICATIONS_NAMESPACE" --dry-run=client -o yaml | oc apply -f - || true
-    kubectl create secret docker-registry redhat-appstudio-registry-pull-secret -n  "$E2E_APPLICATIONS_NAMESPACE" --from-file=.dockerconfigjson=docker.config || true
-    kubectl create secret docker-registry redhat-appstudio-staginguser-pull-secret -n "$E2E_APPLICATIONS_NAMESPACE" --from-file=.dockerconfigjson=docker.config || true
+    kubectl create secret docker-registry redhat-appstudio-user-workload -n $SHARED_SECRET_NAMESPACE --from-file=.dockerconfigjson=docker.config || true
```

Comment:
@flacatus @jkopriva @rhopp @sawood14012 @albarbaro FYI

It's no longer needed to create a pull secret per namespace.

See [this jira epic](https://issues.redhat.com/browse/PLNSRVCE-199) and [this PR](https://github.com/redhat-appstudio/docs/pull/2) for explanation.